exercises/clock/clock.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Clock do
+  defstruct hour: 0, minute: 0
+
+  @doc """
+  Returns a string representation of a clock:
+
+      iex> Clock.new(8, 9) |> to_string
+      "08:09"
+  """
+  @spec new(integer, integer) :: Clock
+  def new(hour, minute) do
+  end
+
+  @doc """
+  Adds two clock times:
+
+      iex> Clock.add(10, 0) |> Clock.add(3) |> to_string
+      "10:03"
+  """
+  @spec add(Clock, integer) :: Clock
+  def add(%Clock{hour: hour, minute: minute}, add_minute) do
+  end
+
+  defimpl String.Chars, for: Clock do
+    def to_string(%Clock{hour: hour, minute: minute}) do
```

Comment:
I like your thinking and normally would totally agree. However the protocol forces the use of a struct so we might as well document it as such.